deploy.sh

```diff
@@ -190,6 +190,9 @@ case "$ZIP_FILENAME" in
     *.tar.gz)
         BUNDLE_TYPE=tgz
         ;;
+    *.tgz)
```

Comment:
Could you bring this up into the previous statement? I believe `*.tar.gz|*.tgz)` is valid.